netmanager/src/redux/DefaultSitePreferences/operations.js

```
@@ -0,0 +1,65 @@
+  setDefaultSitesSuccess,
+  setDefaultSitesFailure,
+  deleteDefaultSiteSuccess,
+  deleteDefaultSiteFailure,
+  updateDefaultSiteSuccess,
+  updateDefaultSiteFailure
+} from './actions';
+
+export const loadDefaultSites = () => async (dispatch) => {
+  try {
+    const response = await getDefaultSelectedSitesApi();
+    if (response && response.selected_sites) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)